python/lsst/afw/cameraGeom/camera.py

```diff
@@ -106,8 +129,38 @@ def findDetectors(self, cameraPoint):
                 detectorList.append(detector)
         return detectorList
 
+    def findDetectorsArray(self, xCoord, yCoord, coordSys):
+        """!Find the detectors that cover a list of points specified by x and y coordinates in any system
+
+        @param[in] xCoord is an array of x coordinates
+        @param[in] yCoord is an array of y coordinates
+        @param[in] coordSys is the coordinate system in which xCoord and yCoord are defined
+        @return a list of lists; each list contains the names of all detectors which contain the
+        corresponding point
+        """
+
```

Comment:
I think we're going to have to start living with some interfaces that accept separate arrays of x and y instead of arrays of points, because arrays of points just aren't NumPy-friendly.

That said, this method apparently uses a vector of Points under the hood (which it populates in a loop in Python), so I can't see how it'd be any more efficient than the interfaces we have now.  If you want to speed it up, I'd think you'd need a C++ interface that takes an `ndarray::Array` for each of x and y, and then loops over them in C++.  Otherwise you're just substituting one loop over points in Python for another loop over the same points in Python.